src/plugins/controls/public/index.ts

```diff
@@ -32,21 +32,21 @@ export {
 } from '../common';
 
 export {
-  ControlGroupContainer,
+  type ControlGroupContainer,
```

Comment:
The `type export` additions in this file: Are they required by a new rule? There is a small chance that we will eventually need to export the actual classes rather than just their types in the future.

Reply:
When running with `isolatedModules` we need strictly type export things if they are not used anywhere in any other form as types.